ghtt/config.py

```diff
@@ -97,7 +102,7 @@ def canonize_group(group):
 
 def get_students(usernames: List[str] = [], groups: List[str] = []) -> List[Person]:
     student_config = get("students", None)
-    return get_persons(student_config, usernames, groups)
+    return sorted(get_persons(student_config, usernames, groups), key=attrgetter('group_nr', 'username'))
```

Comment:
What's the reason for this commit? What happens if the groups don't contain a number? I prefer if we can avoid requirements for what group names should look like.

Reply:
It made a lot more sense for me to handle the Yes/No/Skip/All questions with groups in order. I frequently had to do things for only some specific groups, and the semi-random order was annoying in that case.

But indeed, this needs to be improved, It should handle group names that a number can't be extracted from. I'll fix it and update this PR.

Reply:
I've added https://github.com/IBCNServices/ghtt/pull/1/commits/731fa7fb26b092b11273553b1c24ba63ba43b33b to fix this. (You can always squash this with https://github.com/IBCNServices/ghtt/pull/1/commits/d3e582991d7d4605c52ddaee7715049746e409e3 before merging the PR)